src/Ng/Scripts/Catalog2DnxV3China.cmd

```diff
@@ -0,0 +1,27 @@
+@echo OFF
+	
+cd Ng
+
+:Top
+	echo "Starting job - #{Jobs.ngcatalog2dnxChina.Title}"
+
+	title #{Jobs.ngcatalog2dnxChina.Title}
+	
+    start /w Ng.exe catalog2dnx -source #{Jobs.ngcatalog2dnx.Catalog.Source} ^
+        -contentBaseAddress #{Jobs.ngcatalog2dnxChina.ContentBaseAddress} ^ 
+        -storageBaseAddress #{Jobs.ngcatalog2dnxChina.StorageBaseAddress} ^ 
+        -storageType azure ^ 
+        -storageAccountName #{Jobs.common.v3China.StorageAccountName} ^ 
+        -storageKeyValue #{Jobs.common.v3ChinaStorage.Key} ^ 
```

Comment:
can we use a standard prefix for the variable names? 

`Jobs.Common.China.V3.StorageAccountName`
`Jobs.Common.China.V3.StorageKey`
`Jobs.China.ngCatalog2dns.ContentBaseAddress`
and so on...